codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/ClientSimpleMethods.java

```diff
@@ -0,0 +1,114 @@
+             .map(o -> simpleMethodsTest(o.getValue()))
+             .forEach(builder::addMethod);
+
+        return builder.build();
+    }
+
+    @Override
+    public ClassName className() {
+        return poetExtensions.getClientClass("SimpleMethodsIntegrationTest");
+    }
+
+    /**
+     * Creates a setup method for instantiating a new client. If no regions are present for a service,
+     * us-east-1 will be used. If the service is available in aws-global, that region will be used. If the
+     * service is not available in aws-global but is in us-east-1, that region will be used. If a service is
+     * not available in us-east-1 or aws-global, the first region in the available regions for a service will
+     * be used.
+     */
+    private MethodSpec setup() {
+        ClassName beforeClass = ClassName.get("org.junit", "BeforeClass");
+        ClassName interfaceClass = poetExtensions.getClientClass(model.getMetadata().getSyncInterface());
+        return MethodSpec.methodBuilder("setup")
+                         .addAnnotation(beforeClass)
+                         .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+                         .beginControlFlow("if ($T.serviceMetadata().regions().isEmpty())", interfaceClass)
+                         .addStatement("client = $T.builder().region(Region.US_EAST_1).build()", interfaceClass)
+                         .endControlFlow()
+                         .beginControlFlow("else if ($T.serviceMetadata().regions().contains($T.AWS_GLOBAL))",
+                                           interfaceClass,
+                                           Region.class)
+                         .addStatement("client = $T.builder().region(Region.AWS_GLOBAL).build()", interfaceClass)
+                         .endControlFlow()
+                         .beginControlFlow("else if ($T.serviceMetadata().regions().contains(Region.US_EAST_1))",
+                                           interfaceClass)
+                         .addStatement("client = $T.builder().region(Region.US_EAST_1).build()", interfaceClass)
+                         .endControlFlow()
+                         .beginControlFlow("else")
+                         .addStatement("client = $T.builder().region($T.serviceMetadata().regions().get(0)).build()",
+                                       interfaceClass,
+                                       interfaceClass)
+                         .endControlFlow()
+                         .build();
+    }
+
+    private MethodSpec simpleMethodsTest(OperationModel opModel) {
+        ClassName testClass = ClassName.get("org.junit", "Test");
+        return MethodSpec.methodBuilder(opModel.getMethodName() + "_SimpleMethod_Succeeds")
+                         .addAnnotation(testClass)
+                         .addException(Exception.class)
+                         .addModifiers(Modifier.PUBLIC)
+                         .addStatement("client.$N()", opModel.getMethodName())
```

Comment:
I wouldn't think so. If the service doesn't throw an exception then the call is valid.